lib/engine.js

```diff
@@ -40,6 +43,7 @@ Engine.prototype.start = function(callback) {
   var self = this;
 
   log.info('starting the billing engine');
+  log.info(`connecting to mongoose on ${self._config.storage.mongoURI}`);
```

Comment:
We should make sure that we're not possibly logging credentials here in the Mongo URI.